src/cart/crt_init.c

```diff
@@ -337,17 +337,14 @@ static int data_init(int server, crt_init_options_t *opt)
 
 	d_getenv_uint("D_QUOTA_RPCS", &crt_gdata.cg_rpc_quota);
 
-	/* Must be set on the server when using UCX, will not affect OFI */
+	/* Must be set on the server when using UCX
+         * also on client to allow daos_test/suite system out of dmg to succeed,
+         * will not affect OFI */
 	d_getenv_char("UCX_IB_FORK_INIT", &ucx_ib_fork_init);
-	if (ucx_ib_fork_init) {
-		if (server) {
-			D_INFO("UCX_IB_FORK_INIT was set to %c, setting to n\n", ucx_ib_fork_init);
-		} else {
-			D_INFO("UCX_IB_FORK_INIT was set to %c on client\n", ucx_ib_fork_init);
-		}
-	}
-	if (server)
-		d_setenv("UCX_IB_FORK_INIT", "n", 1);
+	if (ucx_ib_fork_init)
```

Comment:
hmm.. this effectively then allows overwriting of this on engines too, though we scrub all env on engines, so should not be a problem